lib/utils/exceptionHandler.js

```
@@ -0,0 +1,14 @@
+const { SoftError } = require('../errors')
+const logger = require('../logger')
+
+module.exports = (err) => {
+  if (!(err instanceof SoftError)) {
```

Can you make this fail early like:
```
if ((err instanceof SoftError)) process.exit(1)
```